internal/controllers/lvmcluster/resource/vgmanager_daemonset.go

```diff
@@ -234,3 +353,11 @@ func newVGManagerDaemonset(lvmCluster *lvmv1alpha1.LVMCluster, namespace, vgImag
 	setDaemonsetNodeSelector(nodeSelector, &ds)
 	return ds
 }
+
+func getAbsoluteKubeletPath(name string) string {
```

Comment:
Duplicate of https://github.com/openshift/lvm-operator/pull/533/files#diff-ac1e4637145c504870edce40ad17e3f81d5b5d4d5e40fc17b47b1dbe01c212e9R334.